data/remnant/remnant ships.txt

```diff
@@ -1035,6 +1035,60 @@ ship "Puffin"
 
 
 
+ship "Smew"
+	sprite "ship/smew"
+	thumbnail "thumbnail/smew"
+	attributes
+		category "Fighter"
+		licenses
+			Remnant
+		"cost" 1641000
+		"shields" 1200
+		"hull" 2800
+		"cargo space" 90
+		"fuel capacity" 100
+		"ramscoop" 0.8
+		"required crew" 1
+		"bunks" 9
+		"mass" 60
+		"drag" 1.96
```

Comment:
```suggestion
		"drag" 1.75
```

This brings its max speed up a bit to a more reasonable spot that matches the description. (also makes it so that the stock Smew is slightly faster than a stock Boxwing.)